Habitica/res/values/strings.xml

```diff
@@ -953,4 +953,7 @@
     <string name="unlock_previous_short">Finish Quest %d</string>
     <string name="unlock_level_short">Level %d</string>
     <string name="not_participating">You are not participating</string>
+
+    <string name="standard"> standard </string>

```

Comment:
This should be `Standard` with the uppercase first character and without the padding spaces.